utils/creator-utils.ts

```diff
@@ -0,0 +1,17 @@
+import type { ArticlePageProps } from '../pages/artykuly/[slug]';
+
+// Detect YouTube videos by the article URL
+const ytRegex = /youtu(\.be|be\.com)\//g;
+
+// Detect podcasts by the anchor.fm in article URL (most popular hosting) or podcast name (common words like talk or podcast)
+const podcastRegex = /(anchor\.fm\/ | talk | podcast)/i;
```

Comment:
Po co /i?

Reply:
Żeby nie było case-sensitive

Reply:
Ale to coś zmienia? :P a jeśli tak to czemu youtube ma być case sensitive?

Reply:
YouTube może być case sensitive - chodzi o url.

Natomiast podcasty też testujemy pod względem nazwy, no i nie wiesz jak ktoś wpleci w swoją nazwę słowo podcast.